File: docs/user/en/query/filtering/examples/comparable/attribute-in-set.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					attributeInSet(
						"code", 
						"garmin-fenix-6-solar", 
						"garmin-approach-s42-2", 
						"garmin-vivomove-luxe", 
						"garmin-vivomove-luxe-2"
					)
				),
				require(
					entityFetch(
						attributeContent("code")
					)
				)
			)
		);
	}
);

File: documentation/user/en/query/filtering/examples/comparable/attribute-in-set.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		AttributeInSet(
        			"code", 
        			"garmin-fenix-6-solar", 
        			"garmin-approach-s42-2", 
        			"garmin-vivomove-luxe", 
        			"garmin-vivomove-luxe-2"
        		)
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("code")
        		)
        	)
        )
	)
);